tests/Resources/ODS.robot

```diff
@@ -25,14 +25,29 @@ Scale Deployment
     Run    oc -n ${namespace} scale deployment ${deployment-name} --replicas=${replicas}
     Sleep    ${sleep-time}    reason=Wait until ${deployment-name} deployment is scaled to replicas=${replicas}
 
+Scale DeploymentConfig
+    [Documentation]    Sets the size (number of pods) for a deploymentconfig
+    [Arguments]    ${namespace}    ${deploymentconfig-name}    ${replicas}=1    ${sleep-time}=10s
+    Run    oc -n ${namespace} scale deploymentconfig ${deploymentconfig-name} --replicas=${replicas}
+    Sleep    ${sleep-time}    reason=Wait until ${deploymentconfig-name} deploymentconfig is scaled to replicas=${replicas}
+
 Restore Default Deployment Sizes
     [Documentation]    Restores the default sizes to all deployments in ODS
-    ODS.Scale Deployment    redhat-ods-applications    rhods-dashboard    replicas=2
-    ODS.Scale Deployment    redhat-ods-applications    traefik-proxy    replicas=3
-    ODS.Scale Deployment    redhat-ods-monitoring    blackbox-exporter    replicas=1
-    ODS.Scale Deployment    redhat-ods-monitoring    grafana    replicas=2
-    ODS.Scale Deployment    redhat-ods-monitoring    prometheus    replicas=1
-    ODS.Scale Deployment    redhat-ods-operator    rhods-operator    replicas=1    sleep-time=30s
+    [Arguments]    ${restore_deployments}=True    ${restore_deploymentconfigs}=True
+
+    IF  ${restore_deploymentconfigs}==True
+        ODS.Scale DeploymentConfig    redhat-ods-applications    jupyterhub    replicas=3
+        ODS.Scale DeploymentConfig    redhat-ods-applications    jupyterhub-idle-culler    replicas=1
+    END
+
+    IF  ${restore_deployments}==True
+        ODS.Scale Deployment    redhat-ods-applications    rhods-dashboard    replicas=2
+        ODS.Scale Deployment    redhat-ods-applications    traefik-proxy    replicas=3
+        ODS.Scale Deployment    redhat-ods-monitoring    blackbox-exporter    replicas=1
+        ODS.Scale Deployment    redhat-ods-monitoring    grafana    replicas=2
+        ODS.Scale Deployment    redhat-ods-monitoring    prometheus    replicas=1
+        ODS.Scale Deployment    redhat-ods-operator    rhods-operator    replicas=1    sleep-time=30s
+    END
```

Comment:
I did that originally, but having them in just one keyword lets you directly call this from Test Teardown without writing an extra Teardown keyword, like here: https://github.com/red-hat-data-services/ods-ci/blob/3e1bd6792c00821b16cf72ab6067e593a0c04e8a/tests/Tests/200__monitor_and_manage/210__alerts/210__alerts.robot#L141

I think in general it will be just called without using the parameters